pkg/kritis/review/validating_transport_test.go

```diff
@@ -125,6 +125,15 @@ func TestValidatingTransport(t *testing.T) {
 		{name: "invalid attestation authority error", auth: invalidAuthWithOneBadKey, expected: nil, attestations: []metadata.RawAttestation{
 			makeRawAttestationPgp(encodeB64(sig), successFpr),
 		}, errorExpected: true, attError: nil},
+		{name: "auth with generic signature type", auth: validAuthWithOneGoodKey, expected: nil, attestations: []metadata.RawAttestation{
+			makeRawAttestationGeneric("test-sig", "test-id", "generic-address"),
+		}, errorExpected: true, attError: nil},
+		{name: "auth with unknown signature type", auth: validAuthWithOneGoodKey, expected: nil, attestations: []metadata.RawAttestation{
+			{
+				SignatureType: metadata.UnknownSignatureType,
+				Signature:     metadata.RawSignature{},
```

Comment:
Because signature is invalid, this test can not verify that error is triggered on the SignatureType.

Reply:
Done. I gave it a valid signature.